gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,417 @@
+    path: gigamon.app_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Destination IP
+    path: gigamon.dst_ip
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Destination Packets
+    path: gigamon.dst_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: Destination Port
+    path: gigamon.dst_port
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: DNS Flags
+    path: gigamon.dns.flags.rcode
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: DNS ID
+    path: gigamon.dns.id
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: HTTP  User agent OS Family
+    path: gigamon.http.useragent_details.os.family.browser.family
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Http Code
+    path: gigamon.http_code
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: HTTP URL Details
+    path: gigamon.http.url_details.path
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: HTTP User Agent OS Family
+    path: gigamon.http.useragent_details.os.family.device.category
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Http Version
+    path: gigamon.http_version
+    source: log
+    type: double
+  - facetType: list
+    groups:
+      - gigamon
+    name: IP Version
+    path: gigamon.ip_version
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Source IP
+    path: gigamon.src_ip
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Source Packets
+    path: gigamon.src_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: Source Port
+    path: gigamon.src_port
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - gigamon
+    name: SSL Cipher Suite ID
+    path: gigamon.ssl_cipher_suite_id
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: SSL Common Name
+    path: gigamon.ssl_common_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: SSL Issuer
+    path: gigamon.ssl_issuer
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: SSL Validity Not After
+    path: gigamon.ssl_validity_not_after
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: SSL Validity Not Before
+    path: gigamon.ssl_validity_not_before
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - gigamon
+    name: Time Stamp
+    path: gigamon.ts
```

Comment:
This one too, we don't need to have a facet, and the timestamp isn't supposed to be under a Gigamon prefix or facet group since it's a Datadog standard attribute

Reply:
<img width="1027" alt="image" src="https://github.com/user-attachments/assets/2cdcc5ee-7a7b-45b0-afc0-73df112a0d02">
<img width="1026" alt="image" src="https://github.com/user-attachments/assets/217b36f6-4e7f-4225-87aa-6d241bdd5ddf">

Sure, Please confirm the below steps that i did for timestamp. 
1. Created "Date ReMapper" as shown in the screenshot
2. Created "grok parser" as shown in the screenshot.
<img width="1026" alt="image" src="https://github.com/user-attachments/assets/24b686ab-4547-4526-880e-9ec231a09de5">